scripts/hail_batch/hgdp1kg_tobwgs_pca/hgdp_1kg_tob_wgs_pca.py

```diff
@@ -44,15 +45,15 @@ def query(output):  # pylint: disable=too-many-locals
     hgdp1kg_tobwgs_joined = hgdp1kg_tobwgs_joined.annotate_cols(
         hgdp_1kg_metadata=hgdp_1kg_metadata[hgdp1kg_tobwgs_joined.s]
     )
-    mt_path = f'{output}/hgdp1kg_tobwgs_joined_all_samples.mt'
+    mt_path = f'{output}/hgdp1kg_tobwgs_joined_all_samples_densified.mt'
     if not hl.hadoop_exists(mt_path):
         hgdp1kg_tobwgs_joined.write(mt_path)
     hgdp1kg_tobwgs_joined = hl.read_matrix_table(mt_path)
 
     # Perform PCA
-    eigenvalues_path = f'{output}/eigenvalues.csv'
-    scores_path = f'{output}/scores.ht'
-    loadings_path = f'{output}/loadings.ht'
+    eigenvalues_path = f'{output}/eigenvalues_densified.csv'
+    scores_path = f'{output}/scores_densified.ht'
+    loadings_path = f'{output}/loadings_densified.ht'
```

Comment:
Instead of adding an extension here, could we use a `v2` instead of `v1` or similar for the `output` parameter?

Reply:
yup, that makes more sense. I'll go ahead and do that.

Reply:
Is the `web` bucket the best place for this output?

Reply:
The `web` bucket is only for reports that you'd want to serve over a browser, e.g. HTML pages rendered from Rmd or similar.

Reply:
Right now you'd probably want to use `analysis`, but after we've changed the permissions, it would be the `main` bucket.

Reply:
ok perfect. I agree, I think `analysis` is best, but it's currently set to to `web`. I'll put it there